protocol/app/module_accounts_test.go

```diff
@@ -72,6 +74,7 @@ func TestMaccPerms(t *testing.T) {
 		"rewards_vester":         nil,
 		"community_treasury":     nil,
 		"community_vester":       nil,
+		"wasm":                   {"burner"},
```

Comment:
Why does `wasm` need to have burner permission? Can we document?

Reply:
Usually there's no need to explicitly define a module address as a module/

If we do want to add it as a module account, we should explicitly initialize the module account in the upgrade handler to avoid unexpected panic behavior when user send funds to it (relevant [outage](https://www.notion.so/dydx/Module-accounts-not-correctly-initialized-9cbb83d403704f8da8e14617fb499f6f?pvs=4))

Reply:
I saw that Buner permission was already given to cosmwasm in the feature branch, this is just adding a test to it. I'll sync with you on module accounts